androidenhancedvideoplayer/src/main/java/com/profusion/androidenhancedvideoplayer/components/EnhancedVideoPlayer.kt

```diff
@@ -206,6 +216,18 @@ fun EnhancedVideoPlayer(
                             ExoPlayer.REPEAT_MODE_OFF
                         }
                     },
+                    selectedQualityTrack = selectedTrack,
+                    qualityTracks = {
+                        when (val trackSelector = exoPlayer.trackSelector) {
+                            is DefaultTrackSelector ->
+                                trackSelector.generateTrackQualityOptions()
+                            else -> arrayListOf()
```

Comment:
Since this is an expensive calculation, shouldn't we remember it?

Reply:
Remember with a default value to `arrayListOf()`, otherwise you will create a new list every time

Reply:
Done